src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the literal `40` with a constant that represents the daily cost of renting a car to avoid hardcoding values directly into expressions. Use a descriptive name like `DAILY_RENTAL_COST`.